interface/experiment_selection.py

```diff
@@ -0,0 +1,19 @@
+import streamlit as st
```

Comment:
Let me know if you have a WIP branch of the experiment selection not working, I'd be happy to take a look and see if I can figure out why it doesn't play nicely with Streamlit